packages/web-console/src/styles/_grid.scss

```diff
@@ -113,10 +131,11 @@ $visible-drag-handle-margin: 10px;
 .qg-drag-handle {
   // place drag handle so that visible "line" is in the middle of it
   // to add hysteresis for the drag cursor
-  margin-left: -$visible-drag-handle-margin / 2;
-  width: $visible-drag-handle-margin * 2 !important;
+  margin-left: -$visible-drag-handle-margin - $drag-handle-margin;
+  width: $visible-drag-handle-margin * 2 ;
   height: $row-height;
```

Comment:
```suggestion
  width: $visible-drag-handle-margin * 2;
  height: 100%;
  top: 0;
```

Reply:
without this change, the drag handle position is off:

<img width="312" alt="Screenshot 2023-02-13 at 15 12 17" src="https://user-images.githubusercontent.com/4284659/218467401-1965422f-fdb3-4062-941f-907e5fdfb3b7.png">